samples/change-notification/csharp/README.md

```diff
@@ -0,0 +1,89 @@
+﻿### Change Notification sample
+
+Bot Framework v4 ChangeNotification sample.
+
+This sample shows a status of a user in personal scope  and demonstrates presence subscription  lifecycle followed by commands given to Bot.
```

Comment:
`This sample app demonstrates sending change notifications to user presence in Teams based on user presence status`,